src/syntax/specification/specification.ts

```diff
@@ -51,37 +54,49 @@ export function registerElementSpecificationEntry(
         | IElementSpecificationExpression
         | IElementSpecificationStatement
         | IElementSpecificationBlock;
+
+    return name in _elementSpecification;
 }
 
 /**
  * Registers a syntax element specification from a given specification entry table.
  * @param specification specification entry table object with key-value pairs of element name and
  *  corresponding specification entry data
+ * @returns a list of boolean , false if element name already exists else true
```

Comment:
a list of boolean values, \`false\` if element name already exists else \`true\`